css/style.css

```diff
@@ -9,3 +9,89 @@ header {
   background: url("../images/header_bg.jpg") no-repeat center top;
   background-size: cover;
 }
+
+body {
+	font-family: open sans, sans-serif;
+}
+
+#title {
+	color: white;
+	font-size: 20px;
+	font-weight: 800;
+	text-align: left;
+	margin-left: 300px;
+	padding: 20px;
+}
+
+h2 {
+	color: white;
+	font-size: 40px;
+	font-weight: 400;
+	text-align: center;
+	padding: 70px auto;
+}
+
+#getitnow {
+	color: black;
+	font-weight: bold;
+	text-align: center;
+	background-color: #f9e42e;
+	margin: 20px 520px;
+	padding: 10px;
+}
+
+nav {
+	background: white;
+	text-align: left;
+	color: black;
+	font-weight: 300;
+	margin: 50px 50px;
+	padding: 20px;
+}
+
+#benefit {
+	color: black;
+	font-weight: 400;
+	text-align: left;
+	margin-left: 300px;
+	padding: 50px;
+}
+
+#personal {
+	padding: 40px;
+	text-align: left;
+}
+
+section {
+	background: url("../images/testimonial_bg.jpg") no-repeat center top;
+	color: white;
+	text-align: center;
+	padding: 40px;
+}
+
+#quote {
+	text-align: center;
+	font-weight: 300;
+	font-style: italic;
+}
+
+p {
+	font-weight: 600;
+}
+
+#signup {
```

Comment:
Unfortunately, auto is not an allowed value for padding - only margin-left and margin-right. Styles that would improve this are:
```
    background-color: #f9e42e;
    margin: 20px auto;
    padding: 10px;
    width: 200px;
    color: black;
    border-radius: 3px;
```